scripts/policygen/sources/jira/template.yml

```diff
@@ -0,0 +1,42 @@
+source:
+  host: {{ config.jira_domain }}
+  auth: basic_auth:jira
+  rules:
+    - route: "GET /rest/api/3/search"
+      actions:
+      - kind: nullify
+        json_paths:
+        - '$.issues[*].changelog.histories[*].items[?(@.field = "summary")].fromString'
+        - '$.issues[*].changelog.histories[*].items[?(@.field = "summary")].toString'
+        - '$.issues[*].changelog.histories[*].items[?(@.field = "description")].fromString'
+        - '$.issues[*].changelog.histories[*].items[?(@.field = "description")].toString'
+        - $.issues[*].fields.worklog.worklogs[*].comment
+        - $.issues[*].renderedFields.worklog.worklogs[*].comment
+        - $.issues[*].versionedRepresentations
+        - $..description
+      - kind: redact_regex
+        json_paths:
+        - $..summary
+        regex_pattern: '\b(\w{12,}|[\d-]{5,}|\b[A-Za-z0-9._%+-]+@[A-Za-z0-9.-]+\.[A-Z|a-z]{2,}\b)\b'
```

Comment:
This regex is a little hard to understand; could we have defined regexes that are used in different endpoints that can be given a label and specified here so that there's a description of what is being redacted?

Reply:
I'd rather keep it simple and add a comment here.